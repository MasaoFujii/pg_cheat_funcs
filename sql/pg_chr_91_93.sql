\i sql/pg_chr.sql
