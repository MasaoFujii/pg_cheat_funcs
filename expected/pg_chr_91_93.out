\i sql/pg_chr.sql
CREATE EXTENSION pg_cheat_funcs;
\pset null '(null)'
SELECT chr(88) = pg_chr(88);
 ?column? 
----------
 t
(1 row)

SELECT chr(12354) = pg_chr(12354);
 ?column? 
----------
 t
(1 row)

SELECT chr(57000);
 chr 
-----
 íº¨
(1 row)

SELECT pg_chr(57000);
 pg_chr 
--------
 íº¨
(1 row)

SELECT chr(2000000);
 chr 
-----
 
(1 row)

SELECT pg_chr(2000000);
 pg_chr 
--------
 
(1 row)

SELECT pg_chr(code) FROM generate_series(57000, 57010) code;
 pg_chr 
--------
 íº¨
 íº©
 íºª
 íº«
 íº¬
 íº­
 íº®
 íº¯
 íº°
 íº±
 íº²
(11 rows)

DROP EXTENSION pg_cheat_funcs;
